src/components/views/rooms/RoomList.tsx

```diff
@@ -117,9 +118,24 @@ const TAG_AESTHETICS: ITagAestheticsMap = {
         sectionLabel: _td("People"),
         isInvite: false,
         defaultHidden: false,
-        addRoomLabel: _td("Start chat"),
+        addRoomLabel: _td("Add people"),
```

Comment:
Flagging this for design, should this stay as `Start chat` for when the Home space is being viewed?